website/content/en/docs/config/vmtype/ext.md

```diff
@@ -0,0 +1,11 @@
+---
+title: EXT
+weight: 3
+---
+
+> **Warning**
+> "ext" mode is experimental
+
+"ext" option makes use of an external machine, either a virtual machine or a physical machine.
```

Comment:
The idea of "external machines" can be easily confused with "external drivers", so this driver should not be called "ext" ("external").

Let's call this either "generic" or "ssh"

Reply:
I tried the name "ssh", but it looks weird in things like limatype.SSH and so on. So it is not a favorite.